README.rst

```diff
@@ -21,6 +21,22 @@
 
 Get coala to lint all your languages in your project with one tool and config!
 
+Introduction
+------------
+
+coala is a language independent analysis toolkit. It empowers developers
+to create rules which a project's code should conform to. coala takes care
```

Comment:
here, for example, if the name is not bolded, the phrase looks grammatically incorrect at a first glance